apps/gauzy/src/app/pages/approvals/approvals.component.scss

```
@@ -6,20 +6,21 @@
   }
 }
 
+// Approve / Refuse are plain `[nbButton] status="success|danger"` buttons and
```

Stylelint (17.14.1)

[error] Unknown word Approve (CssSyntaxError)

(CssSyntaxError)